src/test/java/io/cdap/plugin/gcp/bigquery/BigQueryRecordToJsonTest.java

```diff
@@ -69,7 +69,7 @@ public void test() throws IOException {
       for (Schema.Field recordField : Objects.requireNonNull(record.getSchema().getFields())) {
```

Comment:
make sure test cases cover the whole code

Reply:
Sure i will add more TC to increase the coverage.